packages/esm-patient-chart-app/src/workspace/workspace-window.component.tsx

```diff
@@ -59,64 +59,58 @@ const WorkspaceWindow: React.FC<ContextWorkspaceParams> = () => {
         className={`${styles.header} ${maximized ? `${styles.fullWidth}` : `${styles.dynamicWidth}`}`}
       >
         {layout === 'tablet' && !canHide && (
-          <Button
-            iconDescription={t('close', 'Close')}
-            hasIconOnly
-            onClick={() => closeWorkspace()}
-            renderIcon={(props) => <ArrowLeft size={16} {...props} />}
-            tooltipPosition="bottom"
-            tooltipAlignment="end"
-          />
+          <IconButton align="bottom-right" onClick={closeWorkspace}>
+            <ArrowLeft />
+          </IconButton>
         )}
         <HeaderName prefix="">{workspaceTitle}</HeaderName>
         <HeaderGlobalBar className={styles.headerGlobalBar}>
           <ExtensionSlot name={patientChartWorkspaceHeaderSlot} />
           {isDesktop(layout) && (
             <>
               {canMaximize && (
-                <Button
-                  iconDescription={maximized ? t('minimize', 'Minimize') : t('maximize', 'Maximize')}
-                  hasIconOnly
+                <IconButton
+                  align="bottom"
                   kind="ghost"
+                  label={maximized ? t('minimize', 'Minimize') : t('maximize', 'Maximize')}
                   onClick={toggleWindowState}
-                  renderIcon={(props) =>
-                    maximized ? <Minimize size={16} {...props} /> : <Maximize size={16} {...props} />
-                  }
-                  tooltipPosition="bottom"
-                />
+                  size="lg"
+                >
+                  {maximized ? <Maximize /> : <Minimize />}
```

Comment:
Thanks @denniskigen for the improvement 🙏 
A small miss, the icons needs to be reverted here
```
{maximized ? <Minimize /> : <Maximize />}
```

I'll create a PR for this

Reply:
Sorry about that. Thanks for catching it, @vasharma05.